docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3'
```

Comment:
Why we not use version 3.7. I used this version and work fine

Reply:
oh sorry, my bad. my docker-compose version can't run version "3.7" in docker-compose.yaml file, so i changed it to 3. I should revert it before push.